### PR TITLE
Irb zh hant

### PR DESCRIPTION
**Pull request by kairstenfay**
Translate the SCAN IRB template into traditional Chinese.

Shares one commit with #16 which I expect will get merged before this PR, where I will then rebase onto master. 

~Translation pending validation by native speakers.~

### CODE REVIEW

**Comment by kairstenfay** [comment]
[SCAN IRB translations - Traditional Chinese](https://trello.com/c/DRgbhIUu/637-scan-irb-translations-traditional-chinese)

File: scan-irb/report-zh-Hant.tex [hunk 1]
```diff
@@ -73,143 +79,127 @@
...
-  \url{http://www.kingcounty.gov/covid}
+  Public Health \textemdash Seattle \& King County（西雅圖和金縣公共衛生）\\
+  \url{https://www.kingcounty.gov/depts/health/covid-19/languages/chinese.aspx}
 
 \item
 
-  Washington State Department of Health\\
-  \url{http://www.doh.wa.gov/Emergencies/Coronavirus}
+  Washington State Department of Health（華盛頓州衛生部）\\
+  \url{www.doh.wa.gov/Emergencies/NovelCoronavirusOutbreak2020COVID19/ChineseSimplified}
```
Comment (kairstenfay) [inline scan-irb/report-zh-Hant.tex]: Regarding this long URL: I'm not sure why line wrapping at the margins works automatically for the longer, following URL but not for this one. e.g. 
image

Comment (kairstenfay) [reply scan-irb/report-zh-Hant.tex]: I resolved this issue here and in https://github.com/seattleflu/lab-result-reports/pull/16 with `\sloppy`

**Comment by kairstenfay** [comment]
This template has been approved by a native speaker.

**Review by joverlee521 — APPROVED** [review]

**Comment by kairstenfay** [comment]
These commits now live at https://github.com/seattleflu/lab-result-reports/pull/18

**Pull request closed without merging**